soh/soh/Enhancements/randomizer/logic.cpp

```diff
@@ -1139,13 +1083,15 @@ namespace Rando {
             case RG_BONGO_BONGO_SOUL:
             case RG_TWINROVA_SOUL:
             case RG_GANON_SOUL:
+            case RG_GREG_RUPEE:
+            case RG_FISHING_POLE:
+                SetRandoInf(RandoGetToRandInf.at(randoGet), state);
+                break;
             case RG_OCARINA_A_BUTTON:
             case RG_OCARINA_C_UP_BUTTON:
             case RG_OCARINA_C_DOWN_BUTTON:
             case RG_OCARINA_C_LEFT_BUTTON:
             case RG_OCARINA_C_RIGHT_BUTTON:
-            case RG_GREG_RUPEE:
-            case RG_FISHING_POLE:
                 SetRandoInf(RandoGetToRandInf.at(randoGet), state);
                 break;
```

Comment:
This change doesn't seem to do anything.

Reply:
That was from an early attempt to preserve helpers that i forgot to revert, will fix